dds_registration/views/membership.py

```diff
@@ -94,7 +94,13 @@ def send_membership_registration_success_message(request: HttpRequest):
             context=context,
             request=request,
         )
-        user.email_user(subject, body, settings.DEFAULT_FROM_EMAIL)
+
+        if invoice.payment_method == "INVOICE" and invoice.status in ("ISSUED", "CREATED"):
+            user.email_user(subject=subject, message=body, from_email=settings.DEFAULT_FROM_EMAIL, attachment_content=create_invoice_pdf(context), attachment_name="DdS Membership Invoice {}.pdf".format(user.get_full_name()))
```

Comment:
There should be removed `from_email` (it's made in PR #106): otherwise, it'll cause an error (because of the order of the optional arguments).

Reply:
No, I was wrong here: the error could be if we mix positional and optional arguments. But I've fixed it anyway.